lib/matplotlib/quiver.py

```diff
@@ -85,6 +85,13 @@
     height of the axes, respectively, when the the window is resized;
     for 'dots' or 'inches', resizing does not change the arrows.
 
+  *orientation*: ['absolute' | 'relative']
```

Comment:
> Probably not many, but there may be some who are using workarounds, which would break if the default behavior were changed.

The question therefore is, are we fixing a bug here? It wouldn't be hard to convince me that the behaviour we currently have is flawed, and that the relative mode simply a bugfix. If the "absolute" mode disappeared altogether I wouldn't lose any sleep over it.

It is always worth remembering the "workflow" xkcd http://xkcd.com/1172/.